websites/R/RealLifeGTAV/dist/metadata.json

```diff
@@ -15,7 +15,7 @@
     "forum.reallifegtav.de",
     "forum.rlv.link"
   ],
-  "version": "2.0.1",
+  "version": "2.1.0",
```

Comment:
2.0.0 xD

Reply:
what no? He did it correct? Now the version is 1 lower...